### ISSUE TITLE
[Suggestion] Show not-yet-spawned Pokemon's time till spawn

### ISSUE THREAD

**Issue by zorakirby** [omkarmoghe/Pokemap#434]
If a pokemon is going to spawn but hasn't spawned yet, it will say "<pokemon> will spawn here, scan again in a while". Is it possible to have it changed to display the time until it spawns? Would be much more convenient instead of scanning the same place every minute until it's there.

**Comment by s7092910** [comment]
The wording we have for it is very confusing I know. But all the pokemon that showup on the map are actively there.

The issue is we don't know when they despawn at this time due to Niantic's API returning an invalid number if they are active for longer than 15 minutes, till they have only 15 minutes left till despawn.

**Comment by zorakirby** [comment]
@s7092910 so you mean like in this example: 
https://puu.sh/qTb8J/1d72a3c9a8.png
the vulpix is actually already there?

**Comment by Anu6is** [comment]
I think they mean, it registers to the system as being there, while it's not actually "active" hence it will not show on the map and it does not has a logical despawn countdown. As such they can't say when it will show, only that it is expected to.

**Comment by s7092910** [comment]
@zorakirby yes I do mean that. But do note that if you haven't scanned it in awhile it might still show up on the map, but not be there.
